applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.h

```diff
@@ -185,6 +193,8 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ElasticIsotropic3D : public C
     ///@name Protected member Variables
     ///@{
 
+    double mStrainEnergy;
```

Comment:
Is it strictly necessary to store the strain energy as a member variable? I'm just asking since this is not a good practice from the memory consumption point of view...